cmd/kubevirt-csi-driver/kubevirt-csi-driver.go

```diff
@@ -21,15 +21,17 @@ import (
 
 var (
 	endpoint               = flag.String("endpoint", "unix:/csi/csi.sock", "CSI endpoint")
-	namespace              = flag.String("namespace", "", "Namespace to run the controllers on")
```

Comment:
ha. that's kind of funny that we have a cli arg that does nothing. Good to see the linter catch stuff like this.